src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1000,6 +1005,21 @@ public void UpdateMicrobePartSelections()
         }
     }
 
+    /// <summary>
+    ///   Updates the visibility of the move cancel button.
+    /// </summary>
+    public void UpdateCancelButtonVisibility()
+    {
+        if (editor.MovingOrganelle == null)
```

Comment:
This should be made into a jsonignore property in the editor class.

Reply:
Wouldn't that make organelle moving not saved?

Reply:
I meant like a property called "CanCancelAction", which dynamically checks if something is happening that could be canceled, so that the GUI doesn't need to know that "hey a move is in progress, and that can be canceled" much easier to reason about the GUI class and its interaction when more responsibility is given to the actual editor class.

Reply:
Ah okay, will look into this.